buildSrc/src/main/kotlin/LocalProtoktBuild.kt

```diff
@@ -21,6 +21,9 @@ import org.gradle.kotlin.dsl.dependencies
 import org.gradle.kotlin.dsl.project
 import org.gradle.kotlin.dsl.withType
 
+val Project.buildSrcClasses
```

Comment:
val -> fun

Reply:
I don't mind making this change. Why do you feel strongly about this?

Reply:
internally, we prefer funs; in this codebase we're not consistent about it

since the value effectively never changes, I am ok leaving it as a val